### ISSUE TITLE
Package request: simplex-chat

### ISSUE THREAD

**Issue by Eloitor** [void-linux/void-packages#45105]
### Package name

simplex-chat

### Package homepage

https://simplex.chat/

### Description

Messaging platform operating without user identifiers of any kind.

There is a cli written in haskell: https://github.com/simplex-chat/simplex-chat/blob/stable/docs/CLI.md#in-any-os

### Does the requested package meet the package requirements?

System, Compiled

### Is the requested package released?

Yes

**Comment by rayfadh** [comment]
Currently working on it, waiting #59323 to get merged first since simplex-chat stable version (v6.4.8) dependency is pinned to ghc v9.6.3. I already have the template for simplex-chat-desktop locally, planning to work on the CLI version soon.

**Comment by classabbyamp** [comment]
we aren't going to accept a template that uses a tool to download a random version of a runtime/compiler, especially because we already have haskell/ghc/etc

**Comment by rayfadh** [comment]
@classabbyamp Genuinely asking, the GHC version on Void is on v9.8.4, how can I make it work when simplex-chat strictly depends on GHC v9.6.3? I wanted to make a PR for it for easy review but I'm planning to wait on my ghcup-hs PR to get merged first.

**Comment by classabbyamp** [comment]
then either ghc has to be updated or the package request can't be fulfilled.

fwiw, simplex is available as a flatpak (`chat.simplex.simplex`) for x86_64 and aarch64

**Comment by hippi777** [comment]
that flatpak relies on an external glibc, static/musl builds were asked a longer while ago without activity, and i think that would be the right way... btw has anyone got a minimalistic bootstrap code for chroot only with glibc without a whole base system? (sounds like easy, but in case anyone already did it...)

they clearly aim for reproducible builds (and they have them on all platforms), so they will most likely stick to those very specific versions of everything, and it is a huge project, so messing with those specific versions for using system packages only asks for trouble... recently i read somewhere that it takes like 20h and 16G+ ram to build the desktop app... :D (maybe one day i will try it with enough swap.....)

nothing promising here either:
https://repology.org/project/simplex-desktop/packages
everyone just use the releases

i think the nearest we could aim for is to make the build work without docker+ubuntu, and i dunno yet how they validate the dependencies, but i guess they do somehow...

otherwise im wondering whether reproducible builds could be accepted from releases as special cases, but anyhow it would be nice to have at least instant info for voiders if nothing else...

(i follow loosely its development, but im yet to try it, and im just learning the languages it uses, and i have no compatible system, void/musl, postmarket (musl) and an ubuntu, but that one on a visionfive (risc-v, which is already asked for, but no progress there either))

**Comment by rayfadh** [comment]
#59353
